lib/msal-browser/src/operatingcontext/NestedAppOperatingContext.ts

```diff
@@ -8,6 +8,12 @@ import { IBridgeProxy } from "../naa/IBridgeProxy";
 import { BridgeProxy } from "../naa/BridgeProxy";
 import { AccountContext } from "../naa/BridgeAccountContext";
 
+declare global {
+    interface Window {
+        __initializeNestedAppAuth?(): Promise<void>;
```

Comment:
nit: Can we please rename to `initializeNestedAppAuth()`? We do not have a specific lint rule yet but for uniformity and since there is no duplication, we can avoid the preceding `__`.

Reply:
I think we informally were sticking to the rule: https://google.github.io/styleguide/tsguide.html#generated-code where __ is used only for system generated JS code. I do not have a strong opinion, however, in the future, if we have a function that does `initializeNestedAppAuth`, that may cause some confusion.

May be we can rename to `__initializeNestedAppAuthRuntime()`?